src/cli/create.ts

```diff
@@ -65,17 +67,25 @@ export async function create (destDir: string | undefined, opts: { template?: st
         destDir = path.resolve(destDir);
     }
 
-    const token = await requireToken();
-    const playpassClient = new PlaypassClient(token);
-    try {
-        await playpassClient.checkGame(subdomain);
-        console.error(`Subdomain ${subdomain}.playpass.games already exists. Please use a different name.`);
-        return;
-    } catch (e) {
-        // Continue
-    }
+    // allow creating the game locally without needing to reserve any remove resources
```

Comment:
comment is a little misleading here, we ought to comment how if this isn't specific to local creation only, we will create resources in our playpass cloud environment